languages/example/irc.lang.json

```diff
@@ -110,6 +119,7 @@
   "Internal error": "Internal error",
   "Invalid CAP subcommand": "Invalid CAP subcommand",
   "Invalid account name": "Invalid account name",
+  "Invalid mode %s parameter: %s": "Invalid mode %s parameter: %s",
```

Comment:
Same deal here, we should make the original string: `"Invalid mode %[1]s parameter: %[2]s"`